src/utils/apiHelpers.ts

```diff
@@ -101,7 +101,7 @@ function externalsToH5pMetaData(obj: any) {
         h5pArray.push({
           copyright: copyrightElement,
           title: i.h5p.title,
-          src: i.h5p.source || '',
+          src: i.url || '',
```

Comment:
I article-converter putter du url inn i h5p-elementet. Blir det ikkje riktig å bruke h5p.url?

Reply:
Nei.. for jeg vil ikke røre det jeg får tilbake fra API'et i tilfellet noe endrer seg der.. så jeg legger det utenfor bare for å være sikker.

Reply:
Så lenge det funker.